src/topology/category/Top/basic.lean

```diff
@@ -16,11 +16,11 @@ universe u
 
 namespace Top
 
-instance topological_space_unbundled (x : Top) : topological_space x := x.str
-
 instance concrete_category_continuous : unbundled_hom @continuous :=
 ⟨@continuous_id, @continuous.comp⟩
 
+instance topological_space_unbundled (x : Top) : topological_space x := x.str
```

Comment:
```suggestion
instance topological_space (x : Top) : topological_space x := x.str
```
This would match the auto-generated name, which is a bit more predictable.

Reply:
Johan, you got distracted from suggesting to change the variable `x` to `X` :)